### DISCUSSION TITLE
AngleSharp v1.0.0.0 or v0.16.1.0?

### DISCUSSION THREAD

**Discussion by mon3104** [Tyrrrz/YoutubeExplode#625] [Q&A] [answered]
So I just migrated to VS2022 community and was updating and compiling everything, had an issue and cleared all dependencies (for a fresh start) and now I cannot figure out why v0.16.1.0 is being downloaded which YoutubeExpole.dll seems to be referencing v1.0.0.0 and causing the run time to fail. Any clues to what I am doing wrong? (I spent the last hour googling with no success)

```
Found conflicts between different versions of "AngleSharp" that could not be resolved.
There was a conflict between "AngleSharp, Version=0.16.1.0, Culture=neutral, PublicKeyToken=e83494dcdc6d31ea" and "AngleSharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=e83494dcdc6d31ea".
"AngleSharp, Version=0.16.1.0, Culture=neutral, PublicKeyToken=e83494dcdc6d31ea" was chosen because it was primary and "AngleSharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=e83494dcdc6d31ea" was not.
References which depend on "AngleSharp, Version=0.16.1.0, Culture=neutral, PublicKeyToken=e83494dcdc6d31ea"
D:\Users\......\packages\AngleSharp.0.16.1\lib\net472\AngleSharp.dll
References which depend on "AngleSharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=e83494dcdc6d31ea" [].
D:\Users\......\packages\YoutubeExplode.6.1.0\lib\net461\YoutubeExplode.dll

```

**Comment by mon3104** [comment, marked as answer]
Stupid me for not figuring this out. Even tough a released version is using an alpha version of another packages (which isn't a good practise).

Anyway if some has this issue as well, simply download manually the alpha version of Angle Sharp v1.0.0.0

**Reply by Tyrrrz** [reply]
Yeah, YoutubeExplode depends on the latest stable AngleSharp, which is 0.16.1 currently.
